huaweicloud/services/acceptance/cdn/data_source_huaweicloud_cdn_quotas_test.go

```diff
@@ -22,7 +22,7 @@ func TestAccDataSourceCdnQuotas_basic(t *testing.T) {
 		ProviderFactories: acceptance.TestAccProviderFactories,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccDataSourceCdnQuotas_basic,
+				Config: testAccDataCdnQuotas_basic,
```

Comment:
```suggestion
				Config: testAccDataQuotas_basic,
```
Do not includes the package name